lib/core/controlplane_api.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 class ControlplaneApi # rubocop:disable Metrics/ClassLength
+  def list_orgs
```

Comment:
Style, I guess `org_list` is more appropriate name (noun vs verb). Plus it will be the same style `gvc_list` below.

Reply:
We need to reach an agreement on what to use, because you prefer "noun + verb", but Justin prefers "verb + noun", so right now all of the method names are very inconsistent. Let's decide on a strategy and apply it in a separate PR.